solr_conf/solrconfig.xml

```diff
@@ -764,8 +764,9 @@
           pub_place
           publisher
           source_id^10
-          content_nostem^50
+          content_nostem^100
           content
+          content_punctuation^100
```

Comment:
I think the punctuation field should have _no_ boosting at all. It seems likely that there's going to be a lot of junk in this field that will give us extraneous matches, so we don't want it boosted.